resources/flaggedrevs.scss

```diff
@@ -1,16 +1,23 @@
+:root {
+	--flaggedrevs-bg-color: #fafafa;
```

Comment:
Why are there fixed now?

Reply:
they were fixed anyway, I have no idea why somebody imported whole theming system to do that